### DISCUSSION TITLE
show live image for focus adjustment

### DISCUSSION THREAD

**Discussion by derjape** [jomjol/AI-on-the-edge-device#1793] [Q&A]
Hi,

is there a way to show a live video stream during setup routine?
this would be very helpful for adjusting the correct focus point.
And it may help while trying to eliminate reflections.

thanks in advance

**Comment by forrestfunk** [comment]
I had the same idea. 
The way I did it now: I flashed a webserver via the arduino IDE. The arduino IDE comes with an example webserver for the cam to test out. I flashed this, put it on my watermeter, adjusted the focus, then I went back and did the usual ai-on-edge installation over the web. 
I had the same idea about somehow implementing this webserver into this project, but my coding skills are not nearly good enough for something like that. The actual developers will know better if that's entirely possible at all with the limited hardware capabilities.

here is a link to a tutorial for the webserver I used. You'll probably find an english tutorial somewhere too if you'd like to do it my way.
https://www.youtube.com/watch?v=0ElLbYCvPNw
